src/amqp.lisp

```diff
@@ -159,11 +159,13 @@
       (t (error "Unexpected error: ~s" reply-type)))))
 
 (defun verify-rpc-framing-call (state channel result)
-  (when (cffi:null-pointer-p result)
-    (verify-rpc-reply state channel (amqp-get-rpc-reply state))))
+  (if (cffi:null-pointer-p result)
+      (verify-rpc-reply state channel (amqp-get-rpc-reply state))
+      t))
 
 (defun maybe-release-buffers (state)
-  (amqp-maybe-release-buffers state))
+  (amqp-maybe-release-buffers state)
+  t)
```

Comment:
The return value of `T` here has the same concern as the previous point. The return value is never used. Is there a reason for this change?

Reply:
Same